lib/chef_apply/text.rb

```diff
@@ -73,5 +83,17 @@ def initialize(path, terminus)
       end
     end
 
+    class MissingPlural < TextError
+      def initialize(path, terminus)
+        set_call_context
+        # Calling back into Text here seems icky, this is an error
+        # that only engineering should see.
+        message = "i18n key #{path}.#{terminus} appears to reference a pluralization.\n"
+        message << "Please append the plural indicator '!!pl' to the end of #{path}.\n"
+        message << "Referenced from #{line}"
```

Comment:
While we expect that only developers will see this error, it's possible that a user will until the testing around text output gets better coverage. What do you think of adding something along the lines of "If you see this error, please open an issue and include this error output!" to crowd-source finding our trouble?